src/katsdpcontroller/generator.py

```diff
@@ -695,7 +752,12 @@ def get_baseline_index(a1, a2):
         SumSensor(sensors, int, f"{stream.name}-xeng-clip-cnt",
                   "Number of visibilities that saturated",
                   name_regex=re.compile(rf"xb\.{stream.name}\.[0-9]+\.xeng-clip-cnt"),
-                  children=stream.n_substreams)
+                  children=stream.n_substreams),
+        SyncSensor(sensors, bool, f"{stream.name}-xengs-synchronised",
+                   "For the latest accumulation, was data present from all F-Engines \
+                   for all X-Engines",
+                   name_regex=re.compile(rf"xb\.{stream.name}\.[0-9]+\.synchronised"),
```

Comment:
Just realised that sense we're pasting a user-provided string into a regex, it probably ought to be escaped (that also applies to the existing SumSensor). Forget the exact syntax, but something like
```suggestion
                   name_regex=re.compile(rf"xb\.{re.escape(stream.name)}\.[0-9]+\.synchronised"),
```
There are limitations on the stream name character set that mean it probably doesn't matter, but better safe than sorry.